### ISSUE TITLE
Enabling external S3 doesn't disable Minio

### ISSUE THREAD

**Issue by mrbazzik** [bentoml/Yatai#272]
When deploying Yatai in EKS and enabling `externalS3` it still uses Minio which claims 16(!) persistent volumes 20G each. There doesn't seem to be a way to control this via values file.

**Comment by mts-dyt** [comment]
This is fixed in version `0.3.14`
```
Installing MinioComponent ...	{"reconciler group": "component.yatai.ai", "reconciler kind": "Deployment", "name": "deployment", "namespace": ""}
minio is not specified, skip installing minio	{"reconciler group": "component.yatai.ai", "reconciler kind": "Deployment", "name": "deployment", "namespace": ""}
Installing DockerRegistryComponent ...	{"reconciler group": "component.yatai.ai", "reconciler kind": "Deployment", "name": "deployment", "namespace": ""}
docker registry is not specified, skip installing docker registry	{"reconciler group": "component.yatai.ai", "reconciler kind": "Deployment", "name": "deployment", "namespace": ""}
```

**Comment by yetone** [comment]
Thanks for your report! As @mts-dyt said, we fixed it in `0.3.14`
